docs/admin/predeployment_check.md

```diff
@@ -266,8 +266,10 @@ will be applied automatically on install).
 
 For non-RPM installations where `daos_server` has been built from source,
 `rmem_default` and `rmem_max` settings should be set to >= 1MB.
-Optionally, the `utils/rpms/daos_server.conf` can be copied to `/etc/sysctl.d/`
+Optionally, the `utils/rpms/10-daos_server.conf` can be copied to `/etc/sysctl.d/`
 to apply the settings automatically on boot.
+Running `sysctl -p /etc/sysctl.d/10-daos_server.conf` will apply these settings
+immediately (prior to a reboot) but will not persist after a reboot.
```

Comment:
"but will not persist after a reboot" isn't true.  Them being copied to /etc/sysctl.d/ as line 269 recommends mean they will be set on every [re]boot.